generated/nifpga/nifpga_service.cpp

```diff
@@ -2856,7 +2856,7 @@ namespace nifpga_grpc {
   NiFpgaFeatureToggles::NiFpgaFeatureToggles(
     const nidevice_grpc::FeatureToggles& feature_toggles)
     : is_enabled(
-        feature_toggles.is_feature_enabled("nifpga", CodeReadiness::kNextRelease))
+        feature_toggles.is_feature_enabled("nifpga", CodeReadiness::kRelease))
```

Comment:
Has all the tests planned to be added to nifpga been created already? I wasn't sure if this was going to have more internal testing with hardware like NI-Sync does.

Reply:
I spoke with Sumit offline, and he is satisfied with the FPGA testing we have conducted so far and the test included as part of this PR. Additionally, we have already [submitted examples](https://github.com/ni/grpc-device/tree/main/examples/nifpga) for the FPGA to help people become familiar with the client code.